### ISSUE TITLE
Skill Swap

### ISSUE THREAD

**Issue by bdjeffyp** [besteon/Ironmon-Tracker#30]
![image](https://user-images.githubusercontent.com/5532354/168370693-d63555cb-a846-4fda-8e62-b2063f6c03a5.png)

Andy let me know of an issue with Skill Swap. I will look up the VOD of the run in question to see if I can get more details. Logging as a bug separate from the Trace ability issue, but there is a chance it is related to the event handler issue we need the newest BizHawk version to address.

**Comment by bdjeffyp** [comment]
Nope, not at all like the Trace issue. Here is the clip link: https://clips.twitch.tv/ObeseAgileCatLeeroyJenkins-X6kXuiP9tfFP0tr7
Skill Swap is a move, not an ability... duh. We don't have the crash log so it may be best to recreate the event and see what the dump looks like.

**Comment by bdjeffyp** [comment]
Interesting... I got a Pokémon with Skill Swap! I used it a couple of time! Nothing bad happened!
First try: https://clips.twitch.tv/ThoughtfulLivelyGrassSwiftRage-emBgUskuwP1btAtu
Second try: https://clips.twitch.tv/GiantSilkyMinkBabyRage-0_CpgHdkr5VPmp4C
No repro... But I do want to try still where the opponent uses the moves rather than the player to see if that will repro the crash.

**Comment by UTDZac** [comment]
Unsure if you still need a clip of this, but saw it happen live today. Starts the trainer battle where the tracker correctly shows the pokemon's ability as Water Absorb. Midway through the battle, Skill Swap is used. After the battle, the Tracker shows Soundproof as the ability. Not shown in clip: later on the Tracker eventually corrects itself back to Water Absorb

https://clips.twitch.tv/SmoothBlightedGoshawkNomNom-XTBl4ZkfjQG1Bpwx
